native/dlaln2.go

```diff
@@ -0,0 +1,386 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+import "math"
+
+// Dlaln2 solves a 1×1 or 2×2 linear system of equations of the form
```

Comment:
Can I just comment how strange it is to have a function that solves a "1x1 system of linear equations". You mean division?